src/Patterns/Path.php

```diff
@@ -322,4 +322,92 @@ private function conditionToString(): string
 
         return sprintf("[%s]", $conditionInner);
     }
+
+    /**
+     * @return MapType|null
+     */
+    public function getProperties(): ?MapType
+    {
+        return $this->properties;
+    }
+
+    /**
+     * Returns the variable of the path.
+     *
+     * @return Variable
+     */
+    public function getVariable(): ?Variable
+    {
+        return $this->variable;
+    }
+
+    /**
+     * Returns the left structure of the relationship.
+     *
+     * @return StructuralType
+     */
+    public function getA(): StructuralType
```

Comment:
I think it would be better to rename "$this->a" to "$this-left" and rename this function to "getLeft"